firebase-firestore/src/main/java/com/google/firebase/Timestamp.java

```diff
@@ -37,7 +38,7 @@
 @PublicApi
 public final class Timestamp implements Comparable<Timestamp>, Parcelable {
 
-  @PublicApi
+  @PublicApi @NonNull
```

Comment:
Newline?

Reply:
I had a newline here but `googleJavaFormat` eated it.

Reply:
How rude!